test/default/vrf

```diff
@@ -0,0 +1,210 @@
+#! /bin/sh
```

Comment:
This file is automatically generated at compile time, it should not be imported in git.